slices/fontconfig-config.yaml

```diff
@@ -0,0 +1,61 @@
+package: fontconfig-config
+
+essential:
+  - fontconfig-config_copyright
+
+slices:
+  config:
+    essential:
+      - fonts-dejavu-core_config
+      - fonts-dejavu-core_fonts
+    contents:
+      /etc/fonts/conf.d/10-hinting-slight.conf:
+      /etc/fonts/conf.d/10-scale-bitmap-fonts.conf:
+      /etc/fonts/conf.d/10-sub-pixel-rgb.conf:
+      /etc/fonts/conf.d/10-yes-antialias.conf:
+      /etc/fonts/conf.d/11-lcdfilter-default.conf:
+      /etc/fonts/conf.d/20-unhint-small-vera.conf:
+      /etc/fonts/conf.d/30-metric-aliases.conf:
+      /etc/fonts/conf.d/40-nonlatin.conf:
+      /etc/fonts/conf.d/45-generic.conf:
+      /etc/fonts/conf.d/45-latin.conf:
+      /etc/fonts/conf.d/48-spacing.conf:
+      /etc/fonts/conf.d/49-sansserif.conf:
+      /etc/fonts/conf.d/50-user.conf:
+      /etc/fonts/conf.d/51-local.conf:
+      /etc/fonts/conf.d/60-generic.conf:
+      /etc/fonts/conf.d/60-latin.conf:
+      /etc/fonts/conf.d/65-fonts-persian.conf:
+      /etc/fonts/conf.d/65-nonlatin.conf:
+      /etc/fonts/conf.d/69-unifont.conf:
+      /etc/fonts/conf.d/70-no-bitmaps-except-emoji.conf:
+      /etc/fonts/conf.d/80-delicious.conf:
+      /etc/fonts/conf.d/90-synthetic.conf:
+      /etc/fonts/fonts.conf:
+      /usr/share/fontconfig/conf.avail/10-hinting-slight.conf:
+      /usr/share/fontconfig/conf.avail/10-scale-bitmap-fonts.conf:
+      /usr/share/fontconfig/conf.avail/10-sub-pixel-rgb.conf:
+      /usr/share/fontconfig/conf.avail/10-yes-antialias.conf:
+      /usr/share/fontconfig/conf.avail/11-lcdfilter-default.conf:
+      /usr/share/fontconfig/conf.avail/20-unhint-small-vera.conf:
+      /usr/share/fontconfig/conf.avail/30-metric-aliases.conf:
+      /usr/share/fontconfig/conf.avail/40-nonlatin.conf:
+      /usr/share/fontconfig/conf.avail/45-generic.conf:
+      /usr/share/fontconfig/conf.avail/45-latin.conf:
+      /usr/share/fontconfig/conf.avail/48-spacing.conf:
+      /usr/share/fontconfig/conf.avail/49-sansserif.conf:
+      /usr/share/fontconfig/conf.avail/50-user.conf:
+      /usr/share/fontconfig/conf.avail/51-local.conf:
+      /usr/share/fontconfig/conf.avail/60-generic.conf:
+      /usr/share/fontconfig/conf.avail/60-latin.conf:
+      /usr/share/fontconfig/conf.avail/65-fonts-persian.conf:
+      /usr/share/fontconfig/conf.avail/65-nonlatin.conf:
+      /usr/share/fontconfig/conf.avail/69-unifont.conf:
+      /usr/share/fontconfig/conf.avail/70-no-bitmaps-except-emoji.conf:
+      /usr/share/fontconfig/conf.avail/80-delicious.conf:
+      /usr/share/fontconfig/conf.avail/90-synthetic.conf:
+      /usr/share/xml/fontconfig/fonts.dtd:
```

Comment:
Do we need this file?

Reply:
The dtd is needed for validating conf files if the xml parser reading them has validation enabled. 
We probably want to leave it in.